client/src/main/scala/org/http4s/client/middleware/RequestLogger.scala

```diff
@@ -26,14 +26,44 @@ object RequestLogger {
       logBody: Boolean,
       redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains,
       logAction: Option[String => F[Unit]] = None
+  )(client: Client[F]): Client[F] =
+    impl[F](logHeaders, Left(logBody), redactHeadersWhen, logAction)(client)
+
+  def logBodyText[F[_]: Concurrent](
```

Comment:
Is this separate from `apply` just for binary compatibility?  I think otherwise `logBody` could have been defaulted to some value of the newer type, but I don't see a way to do that with the existing apply.

Reply:
> Is this separate from apply just for binary compatibility?

Hi @rossabaker - yes, as well as your point in:

> I think otherwise logBody could have been defaulted to some value of the newer type, but I don't see a way to do that with the existing apply.

Also, I did not want to overload `apply` per the up-votes to https://github.com/http4s/http4s/issues/2483#issuecomment-480536534.